app/views/shared/_footer.html.erb

```diff
@@ -45,7 +45,7 @@
 
   <div class="footer__contact bg--grey">
     <div class="row column">
-      <%= link_to root_path, class: 'footer__contact-link' do %>
+      <%= link_to root_path, title: 'Connect home page', class: 'footer__contact-link' do %>
```

Comment:
I spoke to roger about whether the site was GEF Connect or Connect and he said it was Connect on its own. It is only known Internally as 'GEF Connect'